avm/res/network/dnsresolver/main.bicep

```diff
@@ -102,7 +102,7 @@ module dnsResolver_inboundEndpoints 'inbound-endpoint/main.bicep' = [for (inboun
     tags: inboundEndpoint.?tags ?? tags
     location: inboundEndpoint.?location ?? location
     dnsResolverName: dnsResolver.name
-    subnetId: inboundEndpoint.subnetId
+    subnetResourceId: inboundEndpoint.subnetId
```

Comment:
I guess the reference here should also be `subnetResourceId`

Reply:
Could you check in the module overall if it is consistent?

Reply:
Checked all files and module and replaced `virtualNetworkId` with `virtualNetworkResourceId` and `subnetId` with `subnetResourceId`